### DISCUSSION TITLE
增加对Android版本的支持

### DISCUSSION THREAD

**Discussion by amokiel** [asadahimeka/pixiv-viewer#9] [Idea]
总体体验很不错，应用网站推荐都很实用，谢谢推荐。不过移动端用浏览器阅览总感觉不太习惯。希望您可以考虑出个Android版本，这样在移动端体验会更好一些。多谢，辛苦啦！

**Comment by asadahimeka** [comment]
网站支持 PWA 的，可以使用支持 PWA 的浏览器直接安装到桌面，比如 Chrome、Edge或者小米自带的浏览器；
Android 版本的话就是套个 webview，体验上跟 PWA 差不太多，[原版](https://github.com/journey-ad/pixiv-viewer) 是有 apk 安装包的，我有空再看看

**Comment by amokiel** [comment]
> 网站支持 PWA 的，可以使用支持 PWA 的浏览器直接安装到桌面，比如 Chrome、Edge或者小米自带的浏览器； Android 版本的话就是套个 webview，体验上跟 PWA 差不太多，[原版](https://github.com/journey-ad/pixiv-viewer) 是有 apk 安装包的，我有空再看看

PWA是可以安装到桌面，不过移动端总感觉用浏览器滑动起来有点卡顿。如果能够有一个Android版本，那样体验应该更胜一筹，感激不尽。多谢回复。

**Comment by asadahimeka** [comment]
Android 开发不在本人能力范围之内 <img src="https://user-images.githubusercontent.com/31837214/214237746-f3a5420f-9f2e-4eff-b5a4-ce2cb483a6c2.png" width=24>，想要 Android 版的话可以使用 [PixEz](https://github.com/Notsfsssf/pixez-flutter) 哈，我平常都在用这个的

**Comment by amokiel** [comment]
> Android 开发不在本人能力范围之内 <img alt="" width="24" src="https://user-images.githubusercontent.com/31837214/214237746-f3a5420f-9f2e-4eff-b5a4-ce2cb483a6c2.png">，想要 Android 版的话可以使用 [PixEz](https://github.com/Notsfsssf/pixez-flutter) 哈，我平常都在用这个的

还是谢谢你的回复啦。我平时在用Shaft，PixEz，Pixiv Func，Cover这几个。其中Cover应该算是UI最美观的。不过更新有点慢，开发者也是shaft的开发者。这是它的项目链接。https://github.com/CeuiLiSA/Pixiv-Slinky
个人感觉flutter开发的应用体验似乎不是那么丝滑，但也算不上卡顿，不知道怎么回事。像是Shaft和Cover都不是用flutter写的，感觉就流畅了许多。
